MailManager/src/main/java/inc/softserve/health/MailManagerHealthCheck.java

```diff
@@ -31,32 +34,47 @@ public MailManagerHealthCheck(@ESPConnectorAdmin String ESPConnectorAdminUrl,
     }
 
     protected Result check() throws Exception {
-        boolean espConnectorHealthy = false;
-        StringBuilder unhealthyMessage = new StringBuilder("");
+        ExecutorService executor = Executors.newFixedThreadPool(3);
+        FutureTask<Boolean> espConnectorCheck = new FutureTask(() -> isEspConnectorHealthy());
+        FutureTask<Boolean> mongoCheck = new FutureTask(() -> isMongoHealthy());
 
+        FutureTask<Boolean> commonHealth = new FutureTask(() -> {
+            executor.execute(espConnectorCheck);
+            executor.execute(mongoCheck);
+            return espConnectorCheck.get() && mongoCheck.get();
+        });
```

Comment:
My compiler complains about all those three future declatarions. Could you please fix related issue please?